drizzle-orm/src/entity.ts

```diff
@@ -26,16 +26,15 @@ export function is<T extends DrizzleEntityClass<any>>(value: any, type: T): valu
 		);
 	}
 
-	let cls = value.constructor;
-	if (cls) {
-		// Traverse the prototype chain to find the entityKind
-		while (cls) {
-			if (entityKind in cls && cls[entityKind] === type[entityKind]) {
-				return true;
-			}
-
-			cls = Object.getPrototypeOf(cls);
```

Comment:
noticed while reading this code that this was looking for the prototype of the class constructor on each iteration (which should always be Function) rather than looking at the prototype of the current prototype object